File: spec/code_samples/Node_native/list@check_email@{email}/get.js
var http = require("https");

var options = {
  "method": "GET",
  "hostname": "cleanse.studio",
  "port": null,
  "path": "/api/v1/list/check_email/%7Bemail%7D?username=SOME_STRING_VALUE",
  "headers": {
    "content-type": "application/json"
  }
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();

File: spec/code_samples/Python_python3/list@check_email@{email}/get.py
import http.client

conn = http.client.HTTPSConnection("cleanse.studio")

headers = { 'content-type': "application/json" }

conn.request("GET", "/api/v1/list/check_email/%7Bemail%7D?username=SOME_STRING_VALUE", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))